public class CalculatorTest {
    @Test
    public void addは3と4で7を返す() throws Exception {
    }
}
